curated_transformers/models/hf_util.py

```diff
@@ -4,7 +4,17 @@
 
 from .._compat import transformers
 
-SUPPORTED_HF_MODELS = ["roberta-base", "xlm-roberta-base", "xlm-roberta-large"]
+SUPPORTED_BERT_MODELS = ["bert-base-cased"]
```

Comment:
We should also add `bert-base-german-cased` to this list.

Reply:
I made a follow-up PR that will check `model_type` (#9), so that we don't have to enumerate all model names.